src/flag_gems/ops/softmax.py

```diff
@@ -212,9 +212,28 @@ def softmax_kernel_inner(
 
 
 # ------------------------  backward -------------------------------
+@libentry()
+@triton.autotune(
+    configs=[
+        triton.Config({"TILE_K": 4}),
```

Comment:
for performance?

Reply:
to avoid grid error and autotune as well